Snake/SnakeTest/GameSnake_Test.cs

```diff
@@ -0,0 +1,105 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using System.Linq;
+using System.Text;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace SnakeTest
+{
+    [TestClass]
+    public class GameSnake_Test
+    {
+        [TestMethod]
+
+        public void First_Snake_Created_at_Correct_Location()
+        {
+            //create new snake 
+            BusinessLayer.GameSnake testSnake = new BusinessLayer.GameSnake();
+            List<BusinessLayer.Point> testSnakeList;
+            testSnakeList = testSnake.createFirstSnake(40, 40, 3);
+
+
+            //Assert.IsTrue(testSnakeList.);
+            Assert.IsTrue(testSnakeList.Count()==3);
+            // Assert.IsTrue(testSnakeList.in)
+            Assert.IsTrue(testSnakeList.ElementAt(0).returnX() == 20);
+            Assert.IsTrue(testSnakeList.ElementAt(0).returnY() == 20);
+        }
+
+
+        [TestMethod]
+
+        public void Snake_Moves_Right_Correctly()
+        {
+            //create new snake 
+            BusinessLayer.GameSnake testSnake = new BusinessLayer.GameSnake();
+            List<BusinessLayer.Point> testSnakeList;
+            testSnakeList = testSnake.createFirstSnake(20, 40, 3);
+
+            //tested moving right with having eaten food
+            testSnakeList = testSnake.snakeMove(0, true);
+            Assert.IsTrue(testSnakeList.Count() == 4);
+            Assert.IsTrue(testSnakeList.ElementAt(0).returnX() == 11);

```

Comment:
use pascal naming for properties and don't put return prefix it. just use its noun. in this case X would be ok since it is a coordinate